Icons Issue

When I try to install react95/icons using command npm install @react95/icons or yarn add @react95/icons, it doesn't seem to install everything necessary. When I try to follow the instructions in the docs of importing the css file `import '@react95/icons/icons.css'`, I get the error `...node_modules/@react95/icons/esm/react/Access110.js
Module not found: Can't resolve '../../png/Access110_16x16_4.png' in '.../node_modules/@react95/icons/esm/react'`

When I look in the @react95/icons file structure i don't see any png folder with pngs in it. 

Any help would be appreciated! A little more documentation on how to use @react95/icons would also be super helpful.

Thank you so much for making this cool library!

--- a/packages/icons/package.json
+++ b/packages/icons/package.json
@@ -34,6 +34,7 @@
     "build:esm": "cross-env NODE_ENV=esm yarn babel ./src --out-dir dist/esm --extensions .ts,.tsx",
     "build:cjs": "cross-env NODE_ENV=cjs yarn babel ./src --out-dir dist/cjs --extensions .ts,.tsx",
     "build:css": "cp ./icons.css ./dist/icons.css",
+    "postbuild": "cp -r ./png ./dist/png",
     "prepublish": "yarn build",
     "prepublishOnly": "node ../../scripts/prepublish.js --types"
   },

--- a/packages/icons/processIcons.ts
+++ b/packages/icons/processIcons.ts
@@ -27,8 +27,8 @@ const CSS_FILE_PATH = './icons.css';
 
 const makeCssClass = (icon: IconData) => `
 .${icon.id} {
-  width: ${icon.width};
-  height: ${icon.height};
+  width: ${icon.width}px;
+  height: ${icon.height}px;
   background-image: url(${icon.filePath});
 }
 `;

